src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8759,6 +9007,14 @@ definitions:
         properties:
           message:
             type: string
+  DirectAccessUrlBodyCreate:
+  type: object
+  properties:
+    expiresAt:
+      type: string
+      format: date-time
+    validFor:
+      type: string
```

Comment:
validFor should have a type of integer and set a minimum of 1 (see maxItemsParam for an example)